google-cloud-storage/src/test/java/com/google/cloud/storage/BucketInfoTest.java

```diff
@@ -346,6 +351,9 @@ public void testLifecycleRules() {
         setStorageClassLifecycleRule.getAction().getStorageClass());
     assertTrue(setStorageClassLifecycleRule.getCondition().getIsLive());
     assertEquals(10, setStorageClassLifecycleRule.getCondition().getNumNewerVersions().intValue());
+    assertTrue(
```

Comment:
just added these in since I didn't see us testing all the cases of the fromPb switch statement.